src/convertToObject.js

```diff
@@ -1,12 +1,42 @@
 'use strict';
 
+/**
+ * @param {string} sourceString
+ *
+ * @return {object}
+ */
+'use strict';
```

Comment:
The 'use strict' directive is repeated. It should only appear once at the top of the file.